kubernetes/charts/edge-kubernetes/values.yaml

```diff
@@ -69,6 +69,27 @@ iotedged:
     #               1 - All other errors.
     ###############################################################################
     #  maxRetries: 2
+    ###############################################################################
+    # Resource limit settings
+    ###############################################################################
+    # 
+    # We want QoS to be "Guaranteed" so we are setting resource requests and limits
+    # to equal values.  Values can be adjusted here.
```

Comment:
I am not sure having equal values provides guarantees. Contextually requests shall be guaranteed and limits are safety values not to be exceeded. I think it will help also to provide comment on the rational of the default values.

Reply:
I am going by [K8s Quality of Service documentation](https://kubernetes.io/docs/tasks/configure-pod-container/quality-service-pod/#create-a-pod-that-gets-assigned-a-qos-class-of-guaranteed) for that statement. Agreed that I should point to where I derived the values.